src/test/java/cloud/fogbow/ras/core/plugins/interoperability/azure/network/sdk/AzureVirtualNetworkOperationSDKTest.java

```diff
@@ -215,4 +222,196 @@ public void testDoNetworkCreationStepTwoSyncFail() {
         this.azureVirtualNetworkOperationSDK.doNetworkCreationStepTwoSync(indexableExpected, azureCreateVirtualNetworkRef, this.azure);
     }
 
+    // test case: When calling the doGetInstance method with mocked methods
+    // , it must verify if It execute the right AzureGetVirtualNetworkRef.
+    @Test
+    public void testDoGetInstanceSuccessfully() throws FogbowException {
+        // set up
+        String resourceName = "resourceName";
+        String provisioningState = "provisioningState";
+        String id = "id";
+        String name = "name";
+        String cird = "cird";
+
+        VirtualNetworkInner virtualNetworkInner = Mockito.mock(VirtualNetworkInner.class);
+        Mockito.when(virtualNetworkInner.provisioningState()).thenReturn(provisioningState);
+        Mockito.when(virtualNetworkInner.id()).thenReturn(id);
+
+        Network network = Mockito.mock(Network.class);
+        Mockito.when(network.name()).thenReturn(name);
+        Mockito.when(network.inner()).thenReturn(virtualNetworkInner);
+
+        Mockito.doReturn(network)
+                .when(this.azureVirtualNetworkOperationSDK).getNetwork(Mockito.eq(resourceName), Mockito.eq(this.azureUser));
+
+        AzureGetVirtualNetworkRef azureGetVirtualNetworkRefExpected = AzureGetVirtualNetworkRef.builder()
+                .state(provisioningState)
+                .cidr(cird)
+                .name(name)
+                .id(id)
+                .build();
+
+        Mockito.doReturn(cird)
+                .when(this.azureVirtualNetworkOperationSDK).getCIRD(Mockito.eq(network));
+
+        // exercise
+        AzureGetVirtualNetworkRef azureGetVirtualNetworkRef = this.azureVirtualNetworkOperationSDK.doGetInstance(resourceName, this.azureUser);
+
+        // verify
+        Assert.assertEquals(azureGetVirtualNetworkRefExpected, azureGetVirtualNetworkRef);
+    }
+
+    // test case: When calling the doGetInstance method with mocked methods and cird beeing null
```

Comment:
[...] "with mocked methods and cird **beeing** null". Misspelled **being**

Reply:
Done.
Note: I have fixed it in the last branch of Azure Network context.